pkg/kbld/cmd/package.go

```diff
@@ -48,6 +53,12 @@ func NewPackageCmd(o *PackageOptions) *cobra.Command {
 }
 
 func (o *PackageOptions) Run() error {
+	_, err := fmt.Fprintln(os.Stderr, `Command "package" is deprecated, please use 'imgpkg push', learn more in https://carvel.dev/imgpkg/docs/latest/commands/#push`)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintln(os.Stderr, `NOOOOOOOOOOOOOOOO`)
```

Comment:
dont really want this.

Reply:
lol, this was just me testing to ensure the correct commit was being tested :p